docs/design/autoware-architecture/perception/index.md

```diff
@@ -40,8 +40,30 @@ This diagram describes the high-level architecture of the Perception Component.
 
 The Perception Component consists of the following sub-components:
 
-- **Object Recognition**: Recognizes dynamic objects surrounding the ego vehicle in the current frame and predicts their future trajectories.
-- **Obstacle Segmentation**: Identifies point clouds originating from obstacles(not only dynamic objects but also static obstacles that should be avoided, such as stationary obstacles) that the ego vehicle should avoid.
+- **Object Recognition**: Recognizes dynamic objects surrounding the ego vehicle in the current frame, objects that were not present during map creation, and predicts their future trajectories. This includes:
+  - Pedestrians
+  - Cars
+  - Trucks/Buses
+  - Bicycles
+  - Motorcycles
+  - Animals
+  - Traffic Cones
```

Comment:
NITS: It is advisable to standardize the format by either capitalizing the initial letter (Cars) or using lowercase (trees)."